guides/common/modules/proc_enable-insights-with-capsule.adoc

```diff
@@ -0,0 +1,21 @@
+[id='register-capsule-with-insights_{context}']
+
+= Register a Capsule server with Red Hat Insights
+
+Red Hat Insights enables you to proactively diagnose systems and downtime related to security exploits, performance degradation and stability failures.
+You can use the dashboard to quickly identify key risks to stability, security, or performance. 

```

Comment:
In Satellite docs, we followed IBM style guide, so we did not use phrases like "enables you" "allows you" etc, which would be classed as Anthropomorphism according to pg 8 of IBM Style.